helm_deploy/values-prod.yaml

```diff
@@ -11,6 +11,7 @@ generic-service:
     TOKEN_VERIFICATION_API_URL: "https://token-verification-api.prison.service.justice.gov.uk"
     EDUCATION_AND_WORK_PLAN_API_URL: "https://learningandworkprogress-api.hmpps.service.justice.gov.uk"
     PRISONER_SEARCH_API_URL: "https://prisoner-offender-search.prison.service.justice.gov.uk"
+    CURIOUS_API_URL: "https://liveservices.sequation.net/sequation-virtual-campus2-api/"
```

Comment:
Does this mean VC2 and Curious are the same thing?

Reply:
Yes; as I understand it they are one and the same thing